assets/src/styles/blocks/CarouselHeader/CarouselHeaderStyle.scss

```diff
@@ -446,12 +446,14 @@ $medium-image-height: 600px;
     }
 
     p {
+      _-- {
```

Comment:
We might review [this piece of code](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/CarouselHeader/CarouselHeaderStyle.scss#L448-L487). Some styles are wrongly defined through different screens. I don't want to touch the code because is not part of the scope.